common/models/acl.js

```diff
@@ -376,6 +376,15 @@ module.exports = function(ACL) {
   };
 
   /**
+   * Test if ACL permission is not DENY
+   * @returns {Boolean} is permission type different than 'DENY' ?
+   */
+
+  ACL.prototype.isAllowed = function() {
+    return this.permission !== loopback.ACL.DENY;
```

Comment:
This is assuming that the permission value `ACL.DEFAULT` is always `ACL.ALLOW`. I think that's not always true, see L480:

```js
         if (resolved && resolved.permission === ACL.DEFAULT) {
            resolved.permission = (model && model.settings.defaultPermission) || ACL.ALLOW;		            resolved.permission = (model && model.settings.defaultPermission) || ACL.ALLOW;
          }		      
```

Unless I am missing something here?